xonsh/tools.py

```diff
@@ -1472,6 +1473,45 @@ def to_completions_display_value(x):
     return x
 
 
+def is_str_str_dict(x):
+    """Tests if something is a str:str dictionary"""
+    return isinstance(x, dict) and all(
+        isinstance(k, str) and isinstance(v, str) for k, v in x.items()
+    )
+
+
+def to_dict(x):
+    """Converts a string to a dictionary"""
+    if isinstance(x, dict):
+        return x
+    try:
+        x = literal_eval(x)
```

Comment:
And then this needs to be `ast.literal_eval`

Reply:
done already